src/libs/actions/IOU.ts

```diff
@@ -165,6 +165,46 @@ type GPSPoint = {
     long: number;
 };
 
+type RequestMoneyTransactionData = {
+    attendees: Attendee[] | undefined;
+    amount: number;
+    currency: string;
+    comment?: string;
+    receipt?: Receipt;
+    category?: string;
+    tag?: string;
+    taxCode?: string;
+    taxAmount?: number;
+    billable?: boolean;
+    merchant: string;
+    created: string;
+    actionableWhisperReportActionID?: string;
+    linkedTrackedExpenseReportAction?: OnyxTypes.ReportAction;
+    linkedTrackedExpenseReportID?: string;
+};
+
+type RequestMoneyPolicyParams = {
+    policy?: OnyxEntry<OnyxTypes.Policy>;
+    policyTagList?: OnyxEntry<OnyxTypes.PolicyTagLists>;
+    policyCategories?: OnyxEntry<OnyxTypes.PolicyCategories>;
+};
+
+type RequestMoneyParticipantParams = {
+    payeeEmail: string | undefined;
+    payeeAccountID: number;
+    participant: Participant;
+};
+
+type RequestMoneyInformation = {
+    report: OnyxEntry<OnyxTypes.Report>;
+    participantData: RequestMoneyParticipantParams;
```

Comment:
Nice I like how these types are set up now!

Sorry to be picky, but let's change this to participantParams instead to match the type name. Let's do the same for all fields with the suffix "Data". Change the suffix to "Params".